client/src/featureform/resources.py

```diff
@@ -80,6 +80,43 @@ def serialize(self) -> bytes:
         }
         return bytes(json.dumps(config), "utf-8")
 
+@typechecked
+@dataclass
+class AzureBlobStoreConfig:
```

Comment:
Should AzureBlobStoreConfig and OnlineBlobConfig be the same?

Reply:
onlineblobconfig is the config for registering an offline store using a variable blob-compatible store. OnlineBlobConfig could also have an s3 blob or other cloud provider compatible with the backend blob api as a parameter.